MaxText/configs/base.yml

```diff
@@ -168,7 +168,12 @@ per_device_batch_size: 12.0
 expansion_factor_real_data: -1 # if -1 then all hosts will load real data, else total_hosts//expansion_factor_real_data will pull data from GCS.
 eval_per_device_batch_size: 0
 max_corpus_chars: 10_000_000
-dataset_type: c4 # must be c4 or synthetic
+dataset_type: c4 # must be c4, hf or synthetic
+# for HuggingFace input pipeline (dataset_type=hf)
+hf_path: ''
```

Comment:
Could we add some hints here for what this should be set to in comments / maybe a pointer to the nice documentation you wrote in Data Input Pipeline

Reply:
I added more comments and link to the doc